userdata/DynamicConfig.ps1

```diff
@@ -6,46 +6,53 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 Configuration DynamicConfig {
   Import-DscResource -ModuleName PSDesiredStateConfiguration
-
-  Script GpgKeyImport {
-    DependsOn = @('[Script]InstallSupportingModules', '[Script]ExeInstall_GpgForWin')
-    GetScript = { @{ Result = (((Test-Path -Path ('{0}\SysWOW64\config\systemprofile\AppData\Roaming\gnupg\secring.gpg' -f $env:SystemRoot) -ErrorAction SilentlyContinue) -and ((Get-Item ('{0}\SysWOW64\config\systemprofile\AppData\Roaming\gnupg\secring.gpg' -f $env:SystemRoot)).length -gt 0kb)) -or ((Test-Path -Path ('{0}\System32\config\systemprofile\AppData\Roaming\gnupg\secring.gpg' -f $env:SystemRoot) -ErrorAction SilentlyContinue) -and ((Get-Item ('{0}\System32\config\systemprofile\AppData\Roaming\gnupg\secring.gpg' -f $env:SystemRoot)).length -gt 0kb))) } }
-    SetScript = {
-      if ("${env:ProgramFiles(x86)}") {
-        $gpg = ('{0}\GNU\GnuPG\pub\gpg.exe' -f ${env:ProgramFiles(x86)})
-      } else{
-        $gpg = ('{0}\GNU\GnuPG\pub\gpg.exe' -f $env:ProgramFiles)
+  
+  # SourceRepo is in place to toggle between production and testing environments
+	$SourceRepo = mozilla-releng
+	
+  If ($LocationType -eq "AWS") { 
```

Comment:
$LocationType will need to be set here in DynamicConfig as well. Something similar to how it's handled in rundsc.ps1 is fine (if (Get-Service "Ec2Config" -ErrorAction SilentlyContinue){...})